save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -135,7 +136,7 @@ internal class OrganizationController(
             "No organization with name $organizationName was found."
         }
         .filter {
-            organizationPermissionEvaluator.hasGlobalRoleOrOrganizationRole(authentication, it.name, Role.SUPER_ADMIN)
+            organizationPermissionEvaluator.hasGlobalRoleOrOrganizationRole(authentication!!, it.name, Role.SUPER_ADMIN)
```

Comment:
can we property handle nullable here? we can use `Mono.zip(authentication.toMono())`